js/app.js

```diff
@@ -234,54 +281,78 @@ $(document).ready(function () {
+            " due to error: " +
+            errorThrown
+        );
+      },
     });
   }
 
   function clearAllLabels() {
-    $('#form-labels').text('');
-    $('#commit-to-target-repo').text('Commit changes');
-    $('#commit-to-target-repo').attr('disabled', true);
-    $('#commit-to-target-repo').removeClass('btn-success')
-    $('#commit-to-target-repo').addClass('btn-outline-success');
+    $("#form-labels").text("");
+    $("#commit-to-target-repo").text("Commit changes");
+    $("#commit-to-target-repo").attr("disabled", true);
+    $("#commit-to-target-repo").removeClass("btn-success");
+    $("#commit-to-target-repo").addClass("btn-outline-success");
   }
 
   function createNewLabelEntry(label, mode) {
     let action = ' action="none" ';
-    let uncommittedSignClass = '';
+    let uncommittedSignClass = "";
 
-    if (mode === 'copy' || mode === 'new') {
+    if (mode === "copy" || mode === "new") {
       action = ' action="create" new="true" ';
-      uncommittedSignClass = ' uncommitted ';
+      uncommittedSignClass = " uncommitted ";
     }
 
     if (label === undefined || label === null) {
       label = {
-        name: '',
-        color: '',
-        description: ''
+        name: "",
+        color: "",
+        description: "",
       };
     }
 
     let origNameVal = ' data-orig-val="' + label.name + '"';
     let origColorVal = ' data-orig-val="' + label.color + '"';
     let origDescriptionVal = ' data-orig-val="' + label.description + '"';
 
-    let newElementEntry = $('\
-      <div class="label-entry ' + uncommittedSignClass + '" ' + action + '>\
+    let newElementEntry = $(
+      '\
+      <div class="label-entry ' +
+        uncommittedSignClass +
+        '" ' +
+        action +
+        '>\
         <div class="card">\
           <div class="card-body">\
             <div class="flexbox-container">\
-              <input name="name" type="text" class="form-control label-fitting" placeholder="Name" value="' + label.name + '" ' + origNameVal + '>\
-              <input name="color" type="text" class="form-control color-fitting color-box" placeholder="Color"  value="' + label.color + '" ' + origColorVal + '>\
-              <input name="description" type="text" class="form-control description-fitting" placeholder="Description" value="' + label.description + '" ' + origDescriptionVal + '>\
+              <input name="name" type="text" class="form-control label-fitting" placeholder="Name" value="' +
+        label.name +
+        '" ' +
+        origNameVal +
+        '>\
+              <input name="color" type="text" class="form-control color-fitting color-box" placeholder="Color"  value="' +
+        label.color +
+        '" ' +
+        origColorVal +
+        '>\
+              <input name="description" type="text" class="form-control description-fitting" placeholder="Description" value="' +
+        label.description +
+        '" ' +
+        origDescriptionVal +
+        '>\
```

Comment:
Same here. See my comment above.

Reply:
It's formatted by Prettier.

This section itself is very messy. We should rewrite this section between back-ticks `

Reply:
I was also thinking about putting this entire section in back ticks. Are you saying you want to keep it this way for the time being?

Reply:
Yes. 

Because in VS Code, I could manually reformat the code. But when I right-click to format the document. The code above will be reformatted to this state. Therefore, I don't think it is worth our time to fight against formatting extensions/modules.

I believe you use the "format the document" function in VS Code, too.

In the near future, we'll add a formatting module via npm and use the same formatting, so our formatting modules won't fight against each other.

Reply:
Just so you know, I'm also currently using Prettier as my default formatter. I tried formatting `app.js` yesterday and decided to revert the changes solely based on the fact that it does a poor job with HTML code within JS.

Reply:
I think Prettier is innocent. 

I also have another formatter. It yielded a very similar result. 

I think the reason is: those lines are simply too long

Reply:
We should manually break them into multiple lines. 

It shouldn't be difficult to break HTML into multiple lines, as HTML doesn't care much about most line breaks and most spaces.

Reply:
Those lines have > 150 characters in a single line -> too long